Mathlib/Analysis/SpecialFunctions/Pow/Deriv.lean

```diff
@@ -248,6 +262,49 @@ theorem hasDerivAt_ofReal_cpow {x : ℝ} (hx : x ≠ 0) {r : ℂ} (hr : r ≠ -1
     · exact hasDerivAt_id ((-x : ℝ) : ℂ)
     · simp [hx]
 
+/-- An alternate statement of `hasDerivAt_ofReal_cpow`. -/
+theorem hasDerivAt_ofReal_cpow' {x : ℝ} (hx : x ≠ 0) {r : ℂ} (hr : r ≠ 0) :
+    HasDerivAt (fun y : ℝ => (y : ℂ) ^ r) (r * x ^ (r - 1)) x := by
+  have := HasDerivAt.const_mul r <| hasDerivAt_ofReal_cpow hx (by rwa [ne_eq, sub_eq_neg_self])
+  simp_rw [sub_add_cancel, mul_div_cancel₀ _ hr] at this
+  exact this
+
+/-- A version of `DifferentiableAt.cpow_const` for a real function. -/
+theorem DifferentiableAt.cpow_const' {f : ℝ → ℝ} {x : ℝ} (hf : DifferentiableAt ℝ f x)
```

Comment:
Can we put `real` or `ofReal` (as appropriate) into these names instead of priming them?

Reply:
Yes, that's probably a better solution.